larpandoracontent/LArHelpers/LArHierarchyHelper.cc

```diff
@@ -1301,21 +1315,35 @@ void LArHierarchyHelper::MatchInfo::Match()
     {
         MCHierarchy::NodeVector mcNodes;
         m_mcHierarchy.GetFlattenedNodes(pRootMC, mcNodes);
+
+	// Get all of the hits from the MC nodes (for selecting reco hits)
+        CaloHitList allMCHits;
+        for (const MCHierarchy::Node *pMCNode : mcNodes)
+        {
+            const CaloHitList &mcHits{pMCNode->GetCaloHits()};
+            allMCHits.insert(allMCHits.begin(), mcHits.begin(), mcHits.end());
+        }
+
         for (const ParticleFlowObject *const pRootPfo : rootPfos)
         {
             RecoHierarchy::NodeVector recoNodes;
             m_recoHierarchy.GetFlattenedNodes(pRootPfo, recoNodes);
 
             std::sort(mcNodes.begin(), mcNodes.end(),
-                [](const MCHierarchy::Node *lhs, const MCHierarchy::Node *rhs)
+	        [](const MCHierarchy::Node *lhs, const MCHierarchy::Node *rhs)
                 { return lhs->GetCaloHits().size() > rhs->GetCaloHits().size(); });
             std::sort(recoNodes.begin(), recoNodes.end(),
-                [](const RecoHierarchy::Node *lhs, const RecoHierarchy::Node *rhs)
+		[](const RecoHierarchy::Node *lhs, const RecoHierarchy::Node *rhs)
```

Comment:
Looks like this might have been switched to hard tabs, can this be switched back to soft tabs.